com.francescogaglione.cosmicmoney.json

```diff
@@ -0,0 +1,60 @@
+{
+  "id": "com.francescogaglione.cosmicmoney",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "base": "com.system76.Cosmic.BaseApp",
+  "base-version": "stable",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+      "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "command": "cosmic_money",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:ro"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "env": {
+      "CARGO_HOME": "/run/build/cosmic_money/cargo"
+    }
+  },
+  "modules": [
+    {
+      "name": "cosmic_money",
+      "buildsystem": "simple",
+      "build-options": {
+        "env": {
+          "CARGO_HOME": "/run/build/cosmic_money/cargo"
+        }
+      },
+      "build-commands": [
+        "cargo --offline build --release --verbose",
+        "install -Dm0755 ./target/release/cosmic_money /app/bin/cosmic_money",
+        "install -Dm0644 ./res/com.francescogaglione.cosmicmoney.desktop /app/share/applications/com.francescogaglione.cosmicmoney.desktop",
+        "install -Dm0644 ./res/com.francescogaglione.cosmicmoney.metainfo.xml /app/share/metainfo/com.francescogaglione.cosmicmoney.metainfo.xml",
+        "install -Dm0644 ./res/icons/hicolor/scalable/apps/com.francescogaglione.cosmicmoney.png /app/share/icons/hicolor/scalable/apps/com.francescogaglione.cosmicmoney.png",
```

Comment:
if you don't have an actual SVG icon then drop this

Reply:
and fix it upstream too.

Reply:
@bbhtt I don't have an svg, so yes I'll drop it. 
@hfiguiere what do you mean fix it upstream too?

Reply:
`git rm ./res/icons/hicolor/scalable/apps/com.francescogaglione.cosmicmoney.png`

Reply:
@hfiguiere u mean in my repo (cosmic money) or in this repo?